test-app/tests/unit/json-adapter-test.js

```diff
@@ -29,37 +28,30 @@ module(serializer, function (hooks) {
   SharedAdapterBehaviour.mockCreateFailsWithErrorResponse();
 
   module('#mockFindRecord custom', function () {
-    test('when returns json (plain) is used', function (assert) {
-      assert.expect(2);
-      run(() => {
-        let done = assert.async(),
-          json = { id: 1, description: 'the desc' },
-          mock = mockFindRecord('profile').returns({ json }),
-          profileId = mock.get('id');
-
-        FactoryGuy.store
-          .findRecord('profile', profileId)
-          .then(function (profile) {
-            assert.equal(profile.get('id'), profileId);
-            assert.equal(profile.get('description'), json.get('description'));
-            done();
-          });
-      });
+    test('when returns json (plain) is used', async function (assert) {
+      let json = { id: 1, description: 'the desc' },
+        mock = mockFindRecord('profile').returns({ json }),
+        profileId = mock.get('id');
+
+      const profile = await FactoryGuy.store.findRecord('profile', profileId);
+
+      assert.strictEqual(profile.get('id'), profileId.toString());
```

Comment:
`get.id('id')` from a store record produces a string, whereas the id returned by FG is (currently) a number. strictEqual brings this to light, so added toString() where this is relevant.

We'll want to later refactor FG to be strings only. related https://github.com/adopted-ember-addons/ember-data-factory-guy/issues/485